cliquet/tests/test_initialization.py

```diff
@@ -59,6 +59,14 @@ def test_set_the_project_name_from_settings_even_if_specified(self):
         self.assertEqual(config.registry.settings['cliquet.project_name'],
                          'kinto')
 
+    def test_set_version_prefix_redirect_to_false(self):
+        config = Configurator(
+            settings={'cliquet.version_prefix_redirect_enabled': False})
+        cliquet.initialize(config, '0.0.1', 'kinto')
+        redirect_enabled = config.registry.settings[
+            'cliquet.version_prefix_redirect_enabled']
+        self.assertFalse(redirect_enabled)
```

Comment:
You restored the test because of coverage ?

Then it means the other test (disabling the redirect) is not reaching your code properly, otherwise coverage would be 100% without this test.